templates/frontend/pages/submissions.tpl

```diff
@@ -15,20 +15,46 @@
 <div class="page page_submissions">
 	{include file="frontend/components/breadcrumbs.tpl" currentTitleKey="about.submissions"}
 
-	{* Login/register prompt *}
-	{if $isUserLoggedIn}
-		{capture assign="newSubmission"}<a href="{url page="submission" op="wizard"}">{translate key="about.onlineSubmissions.newSubmission"}</a>{/capture}
-		{capture assign="viewSubmissions"}<a href="{url page="submissions"}">{translate key="about.onlineSubmissions.viewSubmissions"}</a>{/capture}
-		<div class="cmp_notification">
-			{translate key="about.onlineSubmissions.submissionActions" newSubmission=$newSubmission viewSubmissions=$viewSubmissions}
-		</div>
-	{else}
-		{capture assign="login"}<a href="{url page="login"}">{translate key="about.onlineSubmissions.login"}</a>{/capture}
-		{capture assign="register"}<a href="{url page="user" op="register"}">{translate key="about.onlineSubmissions.register"}</a>{/capture}
-		<div class="cmp_notification">
-			{translate key="about.onlineSubmissions.registrationRequired" login=$login register=$register}
-		</div>
-	{/if}
+	<div class="submission_sections">
+		<h2>
+			{translate key="section.sections"}
+		</h2>
+		{if $sections|@count > 0}
+			<ul>
+				{foreach from=$sections key="sectionId" item="section"}
+					<li>
+						<h3>
+							{$section.title}
```

Comment:
Section titles should get the `escape` Smarty modifier. (HTML is not expected/permitted there.)

Reply:
Fair enough.

Reply:
Fixed in connection with template redesign in d7afda0 by @NateWr.